### ISSUE TITLE
Unicode translations

### ISSUE THREAD

**Issue by YizhouDing** [wordplaydev/wordplay#320]
Refer to issue #110 

## What's the problem?

Unicode is currently only accessible in English.

## What's the design idea?

Ideally, we would be able to translate 149813 glyphs into all the different languages that Wordplay supports and plans to support. The next steps would be to translate Unicode by categories and push the translations to Wordplay a category at a time (opposed to each translated description at a time so it is less messy for the user searching for glyphs).

## Who benefits?

Non-English users would benefit from this feature.

## Design specification

(This section should be included after a design proposal is ready and approved, and the `buildable` tag is added. This text can remain until then. Designers should add their proposal here, not in a comment).

There is non design spec yet.
